dbt_project.yml

```
@@ -30,5 +23,7 @@ models:
   apple_store:
     materialized: table
     +schema: apple_store
-    intermediate:
+    intermediate:      
```

yml can sometimes be very unhappy with extra spacing. Requesting to remove that to ensure there are no issues.
```suggestion
    intermediate:
```